### PR TITLE
rustc_feature::Features: explain what that 'Option<Symbol>' is about

### PR DESCRIPTION
**Pull request by RalfJung**

### CODE REVIEW

**Comment by jieyouxu** [comment]
Question: is it worth it to introduce a struct with named fields instead of a 3-tuple?

**Comment by RalfJung** [comment]
Possibly... I went for the faster version due to a lack of time, but feel free to take over and do the struct thing :)

File: compiler/rustc_feature/src/unstable.rs [hunk 1]
```diff
@@ -54,6 +56,10 @@ impl Features {
         self.enabled_features.insert(name);
     }
 
+    /// Returns a list of triples with:
+    /// - feature gate name
+    /// - the span of the `#[feature]` attribute
+    /// - (for already stable features) the version since which it is stable
     pub fn enabled_lang_features(&self) -> &Vec<(Symbol, Span, Option<Symbol>)> {
```
Comment (jieyouxu) [inline compiler/rustc_feature/src/unstable.rs:63]: Discussion: not for this PR, but like

```rs
pub struct LangFeature {
    name: Symbol,
    attr_sp: Span,
    stable_since: Option<Symbol>,
}

pub fn enabled_lang_features(&self) -> &Vec<LangFeature> { ... }
```

**Comment by jieyouxu** [comment]
I'll see if it makes sense to package them into a struct.

**Comment by jieyouxu** [comment]
Thanks for the docs!

r? jieyouxu
@bors r+ rollup

File: compiler/rustc_feature/src/unstable.rs (hunk 1 above)
Comment (RalfJung) [reply compiler/rustc_feature/src/unstable.rs:63]: The type should probably be called `EnabledLangFeature`, but other than that -- yeah makes sense.

**Pull request merged**